### PR TITLE
Added issue and PR templates

### PR DESCRIPTION
**Pull request by jaydenwindle**
Addresses issue #41

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here. -->
+
+## How Has This Been Tested?
```
Comment (karimsa) [inline .github/PULL_REQUEST_TEMPLATE.md:10]: Is it worth requiring appropriate unit tests?

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+## Steps to Reproduce (for bugs)
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
+1.
+2.
+3.
+4.
+
+## Context
```
Comment (karimsa) [inline .github/ISSUE_TEMPLATE.md]: I believe context should be covered by "expected/current behaviour".

**Review by karimsa — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (jaydenwindle) [reply .github/ISSUE_TEMPLATE.md]: Ok, sounds good.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (jaydenwindle) [reply .github/PULL_REQUEST_TEMPLATE.md:10]: I think at some point we should implement unit tests, but this section could also apply to manual testing. It's probably important to mention how you tried to test your code, even if it wasn't with legit unit tests.

**Review by karimsa — APPROVED** [review]

**Pull request merged**